src/forms/inputs.js

```diff
@@ -13,25 +13,20 @@ import {
 * @function
 * @param {object} props
 * @property {buttonProps} props.children - React children
-* @property {buttonProps} props.buttonProps - Props to
-*     override any default props that need to be overridden
 *
 * @returns Component
 */
 const BigButton = function ({ children, ...buttonProps }) {
-
-  const overriddenDefaults = {
-    type:      `button`,
-    color:     `teal`,
-    size:      `large`,
-    className: `big-button`,
-    ...buttonProps,
-  };
-
   return (
-    <Button { ...overriddenDefaults }>{ children }</Button>
+    <Button { ...buttonProps }>{ children }</Button>
   );
-};  // End <BigButton>
+};
+BigButton.defaultProps = {
```

Comment:
So many awesome tests! Could you break out these kinds of component changes into separate PRs, though. I'd like to be able to accept them because there are other changes I'd like to make.

Reply:
Yeah, I'll do that.